src/core/Present.jl

```diff
@@ -120,94 +120,98 @@ end
 """
 generator_index(pres::Presentation, x::Symbol) = pres.generator_name_index[x].second
 
-# Presentation macro
-####################
-
-""" Define a presentation using a convenient syntax.
+""" Shorthand for contructing a term in the presentation
 """
-macro present(head, body)
-  name, pres = @match head begin
-    Expr(:call, name::Symbol, syntax_name::Symbol) =>
-      (name, :($(module_ref(:Presentation))($syntax_name)))
-    Expr(:(<:), name::Symbol, parent::Symbol) => (name, :(copy($parent)))
-    _ => throw(ParseError("Ill-formed presentation header $head"))
+function (pres::Presentation)(e::Symbol)
```

Comment:
Let's make this a function with a name rather than making presentation objects callable.